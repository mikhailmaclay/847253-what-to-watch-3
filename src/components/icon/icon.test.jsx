// Libraries
import React from 'react';
import renderer from 'react-test-renderer';
// Components
import Icon from './icon';

describe(`<Icon/>`, () => {
  const props = {
    name: `play-s`,
    width: `20`,
    height: `20`,
  };

  it(`Should render correctly`, () => {
    const result = renderer.create(
        <Icon {...props}/>
    ).toJSON();

    expect(result).toMatchSnapshot();
  });
});
