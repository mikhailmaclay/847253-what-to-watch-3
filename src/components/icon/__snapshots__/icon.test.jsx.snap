// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon/> Should render correctly 1`] = `
<svg
  height="20"
  viewBox="0 0 20 20"
  width="20"
>
  <use
    xlinkHref="#play-s"
  />
</svg>
`;
